src/codeu/chat/common/ConversationHeader.java

```diff
@@ -75,6 +77,8 @@ public ConversationHeader read(InputStream in) throws IOException {
 	// a hashmap with user id as key and their usertype as value
 	private HashMap<Uuid, UserType> userAccessRoles = new HashMap<Uuid, UserType>();
 
+	public LinkedList<String> bots = new LinkedList<String>();
```

Comment:
Priyanka will be making changes to the data structure on her branch to reflect better usability with the server. I arbitrarily chose LinkedList. Don't really know the pros and cons of ArrayList vs LinkedList vs other data structures.